src/app/joint-control/joint-control-core/joint-control-core.component.ts

```diff
@@ -10,20 +10,34 @@ import {MotorService} from "src/app/shared/services/motor.service";
 })
 export class JointControlCoreComponent implements OnInit {
     joint!: JointConfiguration;
-    motorName: string = "";
+    selectedMotor!: any;
+    displayMotors: MotorConfiguration[] = new Array();
 
     constructor(
         private route: ActivatedRoute,
-        private motorService: MotorService,
+        private router: Router,
     ) {}
 
     ngOnInit(): void {
         this.route.data.subscribe((data) => {
-            this.motorName = "";
             this.joint = data["joint"];
+            this.displayMotors = new Array();
+            this.selectedMotor = undefined;
+            this.joint.motors.forEach((motor) => {
+                if (
+                    motor.motorName !== "all_fingers_right" &&
+                    motor.motorName !== "all_fingers_left"
+                ) {
+                    this.displayMotors.push(motor);
+                }
+            });
```

Comment:
can be simplified to:
`this.selectedMotor = undefined;`
` this.displayMotors = this.joint.motors.filter(motor => motor.displaySettings)`